README.md

```diff
@@ -1,18 +1,41 @@
 # retry
 
-An expressive, flexible retry package for Go.
+A small retry package for Go.
```

Comment:
This doesn't really tell me what retry means in this context.

Neither did the old statement, but 🤷‍♀️